client/src/test/java/io/pravega/client/security/auth/NullInitializedTokenProviderImplTest.java

```diff
@@ -0,0 +1,91 @@
+/**
+ * Copyright (c) Dell Inc., or its subsidiaries. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ */
+package io.pravega.client.security.auth;
+
+import io.pravega.client.stream.impl.Controller;
+import lombok.extern.slf4j.Slf4j;
+import org.junit.Test;
+
+import java.time.Instant;
+import java.util.concurrent.CompletableFuture;
+import java.util.function.Supplier;
+
+import static io.pravega.client.security.auth.JwtTestUtils.createJwtBody;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+@Slf4j
+public class NullInitializedTokenProviderImplTest extends JwtTokenProviderImplTest {
```

Comment:
Just because two classes extend one another does not mean their tests have to, or should.

Reply:
I understand that a test class doesn't have to extend another test class just because the objects they test are from classes that form a class hierarchy. The reason for having the `NullInitializedTokenProviderImplTest` test class extend from the `JwtTokenProviderImplTest` test class was that the tests present in latter were also applicable for the former. In any case, now the child test class is removed as `NullInitializedTokenProcviderImpl` is removed too, so this comment is not applicable now.